Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -16,8 +16,8 @@ def update_license_data_args
 
          Update SPDX license data in the Homebrew repository.
       EOS
-      switch "--fail-if-changed",
-             description: "Return a failing status code if current license data's version is different from " \
+      switch "--fail-if-not-changed",
```

Comment:
This makes it the opposite of `brew man` which seems undesirable. Can you explain why they are better being inconsistent?

Reply:
Because, for the purposes of the workflow, I want to make sure I am only detecting when there is a change.

If I make it fail on change then I still can't be certain it has changed as it could have failed for other reasons like the download failing, GitHub API having issues, etc.

Making it a success exit code instead means I can be sure that it has downloaded successfully and it really has detected a change.

The use case of the exit code is differently here than `brew man`. The idea is not to fail the workflow here but rather to check the exit code and push a branch if a change is detected. Accurracy of detection is important if you want to prevent the workflow pushing new branches to the repo without any changes.

Reply:
Ok, makes sense, thanks 👍 . Given that: it might make sense to make `brew man` (maybe even eventually `brew style --fix`?) behave the same and similarly be able to auto-open PRs?

Reply:
Possibly. The only thing is `brew man` CI failures are likely caused by the PR in question (and should be fixed in the same PR), unlike outdated spdx.json data, so any new workflow would perhaps not replace the existing CI step like we did here.

There is some argument however that we should have something that sends `brew man` PRs for commits to Homebrew/bundle, Homebrew/services and Homebrew/test-bot.

Reply:
All true. Probably worth punting on for now, I guess. Thanks!